tests/unit/isStagingDeployLockedTest.ts

```diff
@@ -25,7 +25,7 @@ describe('isStagingDeployLockedTest', () => {
             // Mock the return value of GithubUtils.getStagingDeployCash() to return an empty object
             GithubUtils.getStagingDeployCash = jest.fn().mockResolvedValue({});
             const setOutputMock = jest.spyOn(core, 'setOutput');
-            const isStagingDeployLocked = run();
+            const isStagingDeployLocked = run() as Promise<void>;
```

Comment:
This will result in `Unsafe return of any typed value`, can't be removed yet. What do you think about adding a TODO to remove this once the file is migrated?

Reply:
Yep, sounds good to me 👍

Reply:
Done!